src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```diff
@@ -139,6 +139,14 @@ Overflow menu
   box-shadow: none;
 }
 
+/******************************************
+Page Header
+******************************************/
+
+.header-04 {
+  @include type.type-style('heading-04');
```

Comment:
we can directly add the class like `cds--type-header-04` where you are going to use it right? I think we already do that in several other places.